tests/e2e/transaction/transactionSubmission.test.ts

```diff
@@ -828,4 +830,33 @@ describe("transaction submission", () => {
       expect(submittedTransaction.success).toBe(true);
     });
   });
+
+  describe("transactions with no account authenticator", () => {
+    test("single signer with script payload", async () => {
+      const transaction = await aptos.transaction.build.simple({
+        sender: singleSignerED25519SenderAccount.accountAddress,
+        data: {
+          bytecode: singleSignerScriptBytecode,
+          functionArguments: [new U64(1), receiverAccounts[0].accountAddress],
+        },
+      });
+      try {
+        const authenticator = new AccountAuthenticatorNoAccountAuthenticator();
+        const response = await submitTransaction({
```

Comment:
please follow the other test flows on this file, i.e

```
const transaction = await aptos.transaction.build.simple({...})
const authenticator = aptos.transaction.sign({...})
const response = await aptos.transaction.submit.simple({...})
```

Reply:
This test case may not fit into the usual test flow. It attempts to execute a transaction with `NoAccountAuthenticator` without signing. This should fail because `NoAccountAuthenticator` can be used only in simulation, not in execution. This case intentionally tries to misuse a lower level API such as `submitTransaction`.

Reply:
I understand we dont need to sign, but we should still use `await aptos.transaction.submit.simple({...})` to submit the transaction